Rocks.Wasabee.Mobile.Core.Ui/Views/AgentVerification/SubViews/AgentVerificationStep1View.xaml

```diff
@@ -0,0 +1,78 @@
+      <Frame Padding="0" Margin="10,0" HasShadow="False" CornerRadius="10" BackgroundColor="{DynamicResource BackgroundColorLevel1}">
+          <ScrollView>
+              <Grid RowDefinitions="*, Auto">
+                  <StackLayout Grid.Row="0" VerticalOptions="FillAndExpand" Padding="10">
+                      <Label HorizontalOptions="Center" VerticalOptions="Center" HorizontalTextAlignment="Center" LineHeight="1.25">
+                          <Label.FormattedText>
+                              <FormattedString>
+                                  <Span Text="Welcome to the" FontSize="Large"/>
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="Agent Community Verification" FontSize="22" FontAttributes="Bold"/>
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                              </FormattedString>
+                          </Label.FormattedText>
+                      </Label>
+
+                      <Label Text="🐝     🐝     🐝" FontSize="Large" HorizontalOptions="Center" VerticalOptions="Center" HorizontalTextAlignment="Center" />
+
+                      <Label Margin="0,20,0,0">
+                          <Label.FormattedText>
+                              <FormattedString>
+                                  <Span Text="Your Wasabee account isn't yet verified against Ingress Community forums." />
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="This process is easy and fast to complete." />
+                              </FormattedString>
+                          </Label.FormattedText>
+                      </Label>
+
+                      <Label Text="Step-by-step guide :" FontSize="Medium" Margin="0,20,0,0"/>
+                      <Label Margin="15,0">
+                          <Label.FormattedText>
+                              <FormattedString>
+                                  <Span Text="1. Wasabee generate a secure token (can't be faked or guessed) associated to your in-game name" />
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="2. Log in to the Ingress Community forums and post the token in the Activity feed" />
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="3. Wasabee will verify that the token is posted by the account with the same in-game name." />
+                              </FormattedString>
+                          </Label.FormattedText>
+                      </Label>
+                      <Label Text="This will give Wasabee the proof that you're the owner of both the Wasabee and the Ingress accounts."/>
+                      <Label Text="Attention : This doesn't verify you as an Enlightened player."/>
```

Comment:
here

Reply:
41: "Wasabee generates a secure token (that can't be faked or guessed) to associate with your in-game name."
49: "This will prove to Wasabee that you are the owner of both the Wasabee and Ingress accounts."
50: "Notice: This does not prove that you are an Enlightened Agent, only that you control both accounts."